GitHubLabels.ps1

```diff
@@ -551,7 +552,7 @@ function Set-GitHubLabel
         removed (and thus unassigned from existing Issues) and then the new one created.
 
     .EXAMPLE
-        Set-GitHubLabel -OwnerName Microsoft -RepositoryName PowerShellForGitHub -Label @(@{'name' = 'TestLabel'; 'color' = 'EEEEEE'}, @{'name' = 'critical'; 'color' = 'FF000000'; 'description' = 'Needs immediate attention'})
+        Initialize-GitHubLabel -OwnerName Microsoft -RepositoryName PowerShellForGitHub -Label @(@{'name' = 'TestLabel'; 'color' = 'EEEEEE'}, @{'name' = 'critical'; 'color' = 'FF000000'; 'description' = 'Needs immediate attention'})
```

Comment:
What about `Switch`?

>Specifies an action that alternates between two resources, such as to change between two locations, responsibilities, or states

or `Redo`?

>Resets a resource to the state that was undone

or `Use`?

>Uses or includes a resource to do something

Reply:
I appreciate the suggestions there, but none of those resonate.

`Switch` - Of your suggestions, I think this was the closest, but still I don't think it's as clear as `Initialize`.

`Redo` - implies that there was a previous state, but there isn't necessarily a previous label state for a repo

`Use` - We're not using the labels to do something...we're changing them.

Reply:
I don't want to blow the question out here, but what are the two API endpoints the different functions hit?

If they really have to be different commands with different verbs, I would say:

- `Update` -> `Edit` (that's the approved replacement for `Update`) since it "Modifies existing data by adding or removing content."
- `Set` stays the same, since it "Replaces data on an existing resource or creates a resource that contains some data"

Reply:
The endpoints are identical.  `Initialize` is an old, historical wrapper/helper function that existed before I took the module over.  It calls `Get` to get the current set of labels.  Then it loops through the input array, either adding labels (`New`) if they didn't already exist or updating (`Set`) the colors for labels that already existed, and then deleting (`Remove`) any labels that were previously there but not in the new array.

The functionality doesn't match that of any GitHub provided endpoint...it's just a helper function that had been created by the previous owner.  I'm just trying to name it correctly.

I wanted to avoid changing over to `Edit`, because then I'd be changing _every_ `Set` call over to `Edit` since they all work similarly which would be a _really big_ breaking change unless I aliased-back to `Set-*` as well.  I figured I'd standardize on `Get`/`Set`/`New`/`Remove`, as that all felt rather natural and was mostly what we were already doing.  I'll mull over if I should do the change `Edit` and alias back to `Set`.  I definitely can't migrate to `Edit` without the alias right now.